PowerShell/ScubaGear/baselines/defender.md

```diff
@@ -748,7 +748,21 @@ The alerts SHOULD be sent to a monitored address or incorporated into a SIEM.
 
 4. Select the checkbox next to each alert to enable as determined by the
    agency and at a minimum those referenced in the
-   [_CISA M365 Security Configuration Baseline for Exchange Online_](./exo.md#msexo161v1).
+   [_CISA M365 Security Configuration Baseline for Exchange Online_](./exo.md#msexo161v1) which are:
+
+   a. **Suspicious email sending patterns detected.**
+
+   b. **Suspicious connector activity.**
+
+   c. **Suspicious Email Forwarding Activity.**
```

Comment:
```suggestion
   c. **Suspicious email forwarding activity.**
```

Since everything else is lower case, I would make this lower case to unless there is a compelling reason.

Reply:
Strangely enough that's the name of the alert name, so "technically" it is correct
![Screenshot (125)](https://github.com/cisagov/ScubaGear/assets/149098823/e4de2876-c39d-44f4-b82d-846dde5a3279)